extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1414,6 +1441,11 @@ export class AmpA4A extends AMP.BaseElement {
     if (this.sentinel) {
       mergedAttributes['data-amp-3p-sentinel'] = this.sentinel;
     }
+    const browserSupportsSandbox = this.win.HTMLIFrameElement &&
+        'sandbox' in this.win.HTMLIFrameElement.prototype;
+    if (this.shouldSandbox_ && browserSupportsSandbox) {
+      mergedAttributes['sandbox'] = IFRAME_SANDBOXING_FLAGS.join(' ');
```

Comment:
Why not just make IFRAME_SANDBOXING_FLAGS a string and save the join?

Reply:
Done.